datasets.py

```diff
@@ -142,3 +142,39 @@ def getRatings():
     ratings_df = ratings_df.drop(columns = ['userId', 'timestamp'])
     ratings_df = ratings_df.groupby('movieId', as_index=False).mean()
     return ratings_df
+
+def get_cast():
+    # returns a dataset with the name of the actor/actress, movies they've been in, and characters they've played as
+    credits_df = pd.read_csv("data/credits.csv")
+    credits_dict = {}
+    for row in range(len(credits_df)):
```

Comment:
yeah this is incredibly inefficient looks like it has O(N^2) run time. you can use `.apply(funcName)` here and it will evaluate  each row against the function it will past the row into your function.
```
def cleaningMethod(row):
   cleaningCodeHere


credits_df.apply(cleaningMethod)
```

Reply:
Since this is only run once against a static size dataframe to generate a csv, it is fine. A side note, it will not be O(N^2) run time because all of these single loops only depend on the static length of the df.

Reply:
However, as a future reference, use .apply() but for now it will not hinder speed or performance for our purposes